src/cc/tests/integtest.cc

```diff
@@ -64,7 +64,8 @@ QFSTestEnvironment::TearDown()
 }
 
 void
-QFSTest::init(int numChunkservers) {
+QFSTest::Init(int numChunkservers)
+{
```

Comment:
I think we should move the following line under QFSTest::Init();

MetaserverEnvironment* QFSTest::sMetaserver = new MetaserverEnvironment;